### DISCUSSION TITLE
How do I enable Harvest to collect the metrics  ONTAP File System Analytics capability

### DISCUSSION THREAD

**Discussion by wooyoungAhn** [NetApp/harvest#3044] [Q&A]
Hello
I checked the dashboard below in System Manager.
![image](https://github.com/NetApp/harvest/assets/59076511/65dcdeb6-244c-4be0-8372-b410b243077a)

How do I enable Harvest to collect the metrics below?

REST API: option for custom integration such as Graphana
NetApp® ONTAP® File System Analytics supports REST API
Integrate ONTAP File System Analytics with customer application

Top clients by
/svm​/svms​/{svm.uuid}​/top-metrics​/clients
/storage​/volumes​/{volume.uuid}​/top-metrics​/clients
- iops.read
- iops.write
- throughput.read
- throughput.write

Top directories by
/svm​/svms​/{svm.uuid}​/top-metrics​/directories​
/storage​/volumes​/{volume.uuid}​/top-metrics​/directories
- iops.read
- iops.write
- throughput.read
- throughput.write

Top files by
​/svm​/svms​/{svm.uuid}​/top-metrics​/files
/storage​/volumes​/{volume.uuid}​/top-metrics​/files
- iops.read
- iops.write
- throughput.read
- throughput.write

Top users by
​/svm​/svms​/{svm.uuid}​/top-metrics​/users
/storage​/volumes​/{volume.uuid}​/top-metrics​/users
- iops.read
- iops.write
- throughput.read
- throughput.write

**Comment by rahulguptajss** [comment]
@wooyoungAhn We have two similar issues open for this:

- [Issue #2118](https://github.com/NetApp/harvest/issues/2118)
- [Issue #2591](https://github.com/NetApp/harvest/issues/2591)

Could you add +1 to these issues?
